dset-requirements.txt

```diff
@@ -13,4 +13,9 @@ sncosmo
 wget
 bs4
 tqdm
- 
\ No newline at end of file
+
+# Chandra requirements
+sherpa
+pyvo
```

Comment:
These should have minimum versions associated with them to reduce issues in the future. This can be done through something like: `sherpa>=1.2,<2.0`. (here minimum v1.2 maximum v2.0). Please at least include minimum versions for `sherpa` and `pyvo`.

Reply:
Agreed with @juramaga .